core/src/main/java/com/google/cloud/sql/SqlAdminApiClientFactory.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Let's land this class and the factory as one PR -- and then we can start using it and land all the test changes piece by piece.